src/main/java/eu/europeana/metis/sandbox/service/workflow/ExternalValidationServiceImpl.java

```diff
@@ -1,29 +1,46 @@
 package eu.europeana.metis.sandbox.service.workflow;
 
+import static java.util.Objects.requireNonNull;
+
 import eu.europeana.metis.sandbox.common.exception.RecordProcessingException;
+import eu.europeana.metis.sandbox.common.exception.RecordValidationException;
 import eu.europeana.metis.sandbox.domain.Record;
 import eu.europeana.metis.transformation.service.TransformationException;
+import eu.europeana.validation.service.ValidationExecutionService;
 import org.springframework.stereotype.Service;
 
 @Service
 class ExternalValidationServiceImpl implements ExternalValidationService {
 
-  private OrderingService orderingService;
+  private final static String SCHEMA = "EDM-EXTERNAL";
```

Comment:
"SonarLint: Reorder the modifiers to comply with the Java Language Specification"